packages/loader/container-loader/src/container.ts

```diff
@@ -1061,6 +1061,11 @@ export class Container
 		if (!this.offlineLoadEnabled) {
 			throw new UsageError("Can't get pending local state unless offline load is enabled");
 		}
+		if (this.closed || this._disposed) {
```

Comment:
I think this.closed covers both. Also, is it true that we can't stash if the container is closed but not disposed?

Reply:
they are explicitly different now, but i don't fully understand how. overall i'd rather be defensive here, and not attempt to capture in a closed or deposed state, as they may be due to an error, which mean the components that service this call could be in an undfined state.